packages/__tests__/cost/__snapshots__/registrySnapshots.test.ts.snap

```diff
@@ -414,6 +440,62 @@ exports[`Registry Snapshots endpoint configurations snapshot 1`] = `
       ],
     },
   },
+  "deepseek/deepseek-reasoner": {
```

Comment:
these are coming from previous PR but snapshots were failing, so I had to update them to pass the full CI